workflow/Snakefile

```diff
@@ -18,7 +20,11 @@ rule run_pso_optimization_fmg:
         matlab_function = "main_loop_hpcc_function",
     log:
         "logs/run_pso_fmg_{metadata}.log"
+    resources:
+        time=config["default-resources"]["time"],
+        mem=config["default-resources"]["mem"],
+        cpu_per_task=config["default-resources"]["cpus_per_task"]
```

Comment:
This shouldn't be necessary- the point of default resources is they are the default for all rules